exercises/practice/hello-world/.docs/instructions.hellowordtests.md

```diff
@@ -0,0 +1,228 @@
+## Running Tests Locally
```

Comment:
This file can be deleted. We can use git itself on the main file if we want recover some of its previous contents, no need to have a "backup file".